bindings/nodejs/generated.d.ts

```diff
@@ -265,21 +294,23 @@ export class Operator {
    * await op.write("path/to/file", Buffer.from("hello world"));
    * // or
    * await op.write("path/to/file", "hello world");
+   * // or
+   * await op.write("path/to/file", Buffer.from("hello world"), { contentType: "text/plain" });
    * ```
    */
-  write(path: string, content: Buffer | string): Promise<void>
+  write(path: string, content: Buffer | string, options?: OpWriteOptions | undefined | null): Promise<void>
   /**
    * Write multiple bytes into path.
    *
    * It could be used to write large file in a streaming way.
    */
-  writer(path: string): Promise<Writer>
+  writer(path: string, options?: OpWriteOptions | undefined | null): Promise<Writer>
   /**
    * Write multiple bytes into path synchronously.
    *
    * It could be used to write large file in a streaming way.
    */
-  writerSync(path: string): BlockingWriter
+  writerSync(path: string, options?: OpWriteOptions | undefined | null): BlockingWriter
```

Comment:
Maybe we need `WriterOptions` too?

Reply:
as I mentioned above, I think `WriteOptions` is enough